### ISSUE TITLE
Accessibility - Screen reader is not announcing the selected font and size values

### ISSUE THREAD

**Issue by sryarama** [ckeditor/ckeditor4#4080]
## Type of report

Bug

## Provide detailed reproduction steps (if any)
1. Turn on Voice over
2. Hit the URL https://ckeditor.com/ckeditor-4/demo/
3. Swipe till 'Styles' combo in CK editor.

### Expected result
Screen reader is always announcing pre-defined aria-label. For styles combo it is announcing as "Styles pop button list box popup".

Screen reader user should be able to know which style or font is selected.

### Actual result
Voice over is not announcing the selected Style or font or font size value. Behavior same for other screen readers like JAWS.

## Other details

Browser: Safari (iPad)
Version and Model: 13.3.1 iPad Pro (2nd generation)
OS: iOS
CKEditor version: CKEditor 4.11.4
Installed CKEditor plugins: Font

**Comment by sryarama** [comment]
This issue is present in all screen readers including JAWS.

**Comment by jacekbogdanski** [comment]
I didn't manage to reproduce the issue with JAWS - works fine for me reading correctly combo box labels. However, I can reproduce it with Windows 10 Narrator where the screen reader is unable to read aria-labels, only something similar to **expected** from the issue report.

**Comment by sryarama** [comment]
JAWS is not saying anything about the current selection. I have attached voice memo of JAWS when we focus on Styles combo. When it is focused selected option is "Special Container". I have also attached the screenshot for more clarity.

![image](https://user-images.githubusercontent.com/66038900/83473979-ba142a00-a43f-11ea-9278-6f915436ed7e.png)

[voicememo.zip](https://github.com/ckeditor/ckeditor4/files/4714423/voicememo.zip)

**Comment by sryarama** [comment]
> I didn't manage to reproduce the issue with JAWS - works fine for me reading correctly combo box labels. However, I can reproduce it with Windows 10 Narrator where the screen reader is unable to read aria-labels, only something similar to **expected** from the issue report.

JAWS is not saying anything about the current selection. I have attached voice memo of JAWS when we focus on Styles combo. When it is focused selected option is "Special Container". I have also attached the screenshot for more clarity.

![image](https://user-images.githubusercontent.com/66038900/83474126-170fe000-a440-11ea-826f-455315397136.png)
[voicememo.zip](https://github.com/ckeditor/ckeditor4/files/4714444/voicememo.zip)

**Comment by sryarama** [comment]
Hey, is there any update on this? It is impacting our accessibility compliance.

**Comment by jacekbogdanski** [comment]
Hey @sryarama, so far we have no progress on this issue due to other priorities in CKEditor 4.

If you looking for a way to push this issue forward, you may contact our [support team](https://ckeditor.com/contact/) to see if there are any options to speed things up.

**Comment by Comandeer** [comment]
The issue was fixed in #4580. After the fix, the dropdown value is announced right before its name.

**Comment by sryarama** [comment]
As I mentioned in the description, issue was reported in iOS and Voice over. When voice over is on and tap on rich combo/drop down, the focus is not staying on the rich combo/dropdown instead focus jumps to the content area.

I agree that label was added and working fine with other screen readers but rich combos are no longer usable with voice over.

**Comment by jacekbogdanski** [comment]
@sryarama I think you may be referring to another issue with VoiceOver reported in https://github.com/ckeditor/ckeditor4/issues/4855 that is still open.
